lib/importer/factory/object_factory.rb

```diff
@@ -0,0 +1,132 @@
+require 'importer/log_subscriber'
+module Importer
+  module Factory
+    class ObjectFactory
+      extend ActiveModel::Callbacks
+      define_model_callbacks :save, :create
+      after_save :attach_files
+
+      class_attribute :klass, :attach_files_service, :system_identifier_field
+
+      attr_reader :attributes, :files_directory, :object, :files
+
+      def initialize(attributes, files_dir = nil, files = [])
+        @files_directory = files_dir
+        @files = files
+        @attributes = attributes
+      end
+
+      # rubocop:disable Metrics/MethodLength
```

Comment:
This is much better because rubocop tells you when you've resolved them and that you should remove the exemption.  It also shows people where the code has the problem, as opposed to rubocop_todo.yml